pkg/service/roomservice.go

```diff
@@ -62,19 +62,12 @@ func (s *RoomService) DeleteRoom(ctx context.Context, req *livekit.DeleteRoomReq
 	if err := EnsureCreatePermission(ctx); err != nil {
 		return nil, twirpAuthError(err)
 	}
-	// if the room is currently active, RTC node needs to disconnect clients
-	node, err := s.router.GetNodeForRoom(ctx, req.Room)
-	if err != nil {
-		return nil, err
-	}
 
-	err = s.router.WriteRTCNodeMessage(ctx, node.Id, &livekit.RTCNodeMessage{
-		ParticipantKey: s.roomParticipantKey(req.Room),
+	if err := s.writeRoomMessage(ctx, req.Room, "", &livekit.RTCNodeMessage{
```

Comment:
With a blank participantID, this is a room message instead of a node message so that `EnsureAdminPermission` gets checked?